test/functions.jl

```diff
@@ -44,6 +44,39 @@ nda = NamedDimsArray(a, (:x, :y))
         end
     end
 
+    @testset "eachslice" begin
+        if VERSION > v"1.1-"
```

Comment:
Do you think we should delcare `nda` within this testset?
This file is getting a buit long to have to read up to check what it was.

Reply:
Good suggestion -- done!